### PR TITLE
Improve docs for filters

### PR DESCRIPTION
**Pull request by forresst**
* add filters `dump` and `sum`
* Sort list by alphabetical order

### CODE REVIEW

File: docs/templating.md [hunk 1]
```diff
@@ -792,6 +792,11 @@ If `preserve_linebreaks` is true, normalizes whitespace, trying to preserve orig
 linebreaks. Use second behavior if you want to pipe
 `{{ text | striptags | nl2br }}`. Use default one otherwise.
 
+### dump (object)
+
+Mainly debugging, it's possible to see what's available on an object, it is pretty useful. `dump` is a builtin
```
Comment (carljm) [inline docs/templating.md]: Suggested re-wording:

> Call `JSON.stringify` on an object and dump the result into the template. Useful for debugging: `{{ foo | dump }}`.

**Comment by carljm** [comment]
Thanks! I made a suggestion for a shorter wording of the English docs for `dump` -- the French version is up to you of course :-)

**Comment by forresst** [comment]
@carljm No soucy, thanks for your help ! i will do tomorrow

**Comment by forresst** [comment]
@carljm Done

**Comment by carljm** [comment]
Thanks! Merged.

**Comment by carljm** [comment]
Hi @forresst - I've added you as a repo collaborator, so in the future for any PR that's just about updating the French docs, you can just commit it directly yourself. I'd prefer if you don't commit changes to `src/*` without getting a review from another maintainer first (I do the same myself).

**Comment by forresst** [comment]
Hi @carljm - I would use it only for doc in French. In the case of an addition of a doc in English, I will continue to do PR for a review. And of course, always a PR for the code.

Thank you for your confidence.
